lib/modules/bulk_data/bulk_data_authorization_sequence.rb

```diff
@@ -0,0 +1,269 @@
+# frozen_string_literal: true
+
+module Inferno
+  module Sequence
+    class BulkDataAuthorizationSequence < SequenceBase
+      title 'Bulk Data Authentication'
+
+      test_id_prefix 'BDA'
+
+      requires :bulk_client_id, :bulk_private_key, :bulk_token_endpoint
+
+      description 'Test Bulk Data Authorization Token Endpoint'
+
+      def authorize(bulk_private_key: @instance.bulk_private_key,
+                    content_type: 'application/x-www-form-urlencoded',
+                    scope: 'system/*.read',
+                    grant_type: 'client_credentials',
+                    client_assertion_type: 'urn:ietf:params:oauth:client-assertion-type:jwt-bearer',
+                    iss: @instance.bulk_client_id,
+                    sub: @instance.bulk_client_id,
+                    aud: @instance.bulk_token_endpoint,
+                    exp: 5.minutes.from_now,
+                    jti: SecureRandom.hex(32))
+        id_token = JSON::JWT.new(
```

Comment:
Maybe just call this `token`? `id_token` makes it seem like it's OIDC related.

Reply:
renamed to jwt_token